lib/data-storage.js

```diff
@@ -59,7 +59,7 @@ class DataStorage {
   putData(dataType, data, context = null) {
     if (!dataType || !data) return;
 
-    context = context || this.context || global.testomatioTestTitle || jestHelpers.getIdOfCurrentlyRunningTest();
+    context = context || this.context || global.testomatioTestTitle || jestHelpers.getNameOfCurrentlyRunningTest();
```

Comment:
```suggestion
    context = context || this.context || testrRunnnerHelper.getNameOfCurrentlyRunningTest();
```

global.testomatioTestTitle move to `testrRunnnerHelper`